build/release/Makefile

```
@@ -25,6 +25,13 @@ ifeq ($(filter master release,$(CHANNEL)),)
 $(error invalid channel $(CHANNEL))
 endif
 
+# When a release branch is first created, the tag shows up in the master branch build tags.
+# When that tag has the alpha, beta, or rc suffixes, we want to remove it since it is only
+# relevant to the release tags.
+ifeq ($(CHANNEL),master)
+override VERSION := $(shell echo "$(VERSION)" | sed -e 's/-alpha.0//' | sed -e 's/-beta.0//' | sed -e 's/-rc.0//')
```

```suggestion
override VERSION := $(shell echo "$(VERSION)" | sed -e 's/-alpha.0//' -e 's/-beta.0//' -e 's/-rc.0//')
```

Nit: you can pass `-e` as many times as you want instead of piping.

---

Travis, didn't someone mention not to do this in the docs because `-e` doesn't exist for `sed` on mac?

---

This `sed` command with multiple `-e` arguments was working on my mac just now. Now that you mention it, I remember having an issue with some command in the upgrade guide. Do you recall which command that was?

---

oh I think it was an issue with `sed -i` rather than `sed -e`

---

Yeah it was `-i` not `-e` :-)